src/components/InForm/InForm.tsx

```diff
@@ -32,42 +33,51 @@ export const InForm: React.FunctionComponent<any> = (props) => {
   const [formData, setFormData] = useState<IInForm>(defaultInForm)
   const [gradeRankOptions, setGradeRankOptions] = React.useState<IComboBoxOption[]>([]);
 
-  const onEmpTypeChange = React.useCallback((ev: React.SyntheticEvent<HTMLElement> | undefined, option?: IChoiceGroupOption) => {
-    if (option)
-      setFormData((f: IInForm) => {
-        switch (option.key) {
-          case "civ":
-            setGradeRankOptions([...GS_GRADES, ...NH_GRADES]);
-            break;
-          case "mil":
-            setGradeRankOptions([...MIL_GRADES]);
-            break;
-          case "ctr":
-            setGradeRankOptions([]);
-            break;
-        }
-        return { ...f, empType: option.key as emptype }
-      });
+
+  const empNameId = useId('empName');
+  const localRemoteId = useId('localRemote');
+  const newCivId = useId('newCiv')
+  const empTypeId = useId('empType')
+  const gradeRankId = useId('gradeRank')
+  const officeId = useId('office')
+  const arrivalDateId = useId('arrivalDate')
+  const supervisorId = useId('supervisor')
+
+  const onEmpTypeChange = React.useCallback((ev: FormEvent<HTMLElement>, data: RadioGroupOnChangeData) => {
+    setFormData((f: IInForm) => {
+      switch (data.value) {
+        case "civ":
+          setGradeRankOptions([...GS_GRADES, ...NH_GRADES]);
+          break;
+        case "mil":
+          setGradeRankOptions([...MIL_GRADES]);
+          break;
+        case "ctr":
+          setGradeRankOptions([]);
+          break;
+      }
+      return { ...f, empType: data.value as emptype }
+    });
   }, []);
 
-  const onLocalRemote = React.useCallback((ev: React.FormEvent<IToggle>, checked: boolean | undefined) => {
+  const onLocalRemote = React.useCallback((ev: ChangeEvent<HTMLElement>, data: SwitchOnChangeData) => {
```

Comment:
I don't think we need the useCallback on these since we're not memoizing these forms. Doesn't really hurt things, but adds unnecessary code complexity.
63, 69, 75, 85, 95

Reply:
Agree, used this from a sample.  Did some reading later on a debate between a team on whether you should always do the useCallaback, and for these simple cases the useCallback does have a higher compute time then the potential savings because of the number of fields, and what the function is doing.